### DISCUSSION TITLE
FS22_techmagri12M32T

### DISCUSSION THREAD

**Discussion by sschu74** [ddewar/FS22_UniversalAutoloadModhubAddon#112] [Q&A] [answered]
I was able to get FS22_techmagri12M32T working,  using FS22_lizard_rp03 settings as starting point.   However I have a problem with pallet load order.  They load in full stacks towards front of the trailer first working towards rear all in full stacks.

	<vehicleConfiguration configFileName="FS22_techmagri12M32T/techmagriTransTruck.xml">
	    <loadingArea offset="0 1.25 -1.03" width="2.5" height="2.6" length="12.15" baleHeight="3.8"/>
	    <options isBaleTrailer="false" enableRearLoading="true" enableSideLoading="true" showDebug="false"/>
	</vehicleConfiguration>

rp03 doesn't do this.    What controls the order of pallet loading location to make work like the rp03 does ?

**Comment by ddewar** [comment, marked as answer]
Hey @sschu74, I believe what you're looking for is called horizontal loading. You can either add it to your config ```horizontalLoading="true"```

Or there might be a keybind in Universal Autoload to toggle the loading method. Haven't looked in a while so I cannot be sure. Let me know if that fixes it.

**Comment by sschu74** [comment]
adding horizontalLoading="true" had no impact for me but keybind fixed it.  That is actually better being able to switch in game.
I should have caught that option , thank you !
